app/screens/security-screen/two-factor-authentication-selection-screen.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react"
+import { useCallback, useEffect, useState } from "react"
+import { gql, useMutation } from "@apollo/client"
+import { Linking, Text, View } from "react-native"
+import { ListItem } from "react-native-elements"
+import EStyleSheet from "react-native-extended-stylesheet"
+import { StackNavigationProp } from "@react-navigation/stack"
+
+import { Screen } from "../../components/screen"
+import { color } from "../../theme"
+import { translate } from "../../i18n/translate"
+import { RootStackParamList } from "../../navigation/stack-param-lists"
+import { toastShow } from "../../utils/toast"
+import { TwoFAVerificationType } from "."
+
+const GENERATE_2FA = gql`
+  mutation generate2FA {
+    generate2fa {
+      secret
+      uri
+    }
+  }
+`
+
+// const GENERATE_2FA = gql`
+//   mutation twoFAGenerate {
+//     twoFAGenerate {
+//       errors {
+//         message
+//       }
+//       twoFASecret {
+//         secret
+//         uri
+//       }
+//     }
+//   }
+// `
```

Comment:
Do we need this?

Reply:
This is for switching things over to api v2. I was having some trouble with some of the mutations not being ready when I was working on it.